mreg_cli/config.py

```diff
@@ -38,7 +41,19 @@
     )
 )
 
-DEFAULT_LOG_FILE = os.path.expanduser("~/.mreg-cli.log")
+data_dir = platformdirs.user_data_dir(appname="mreg-cli", appauthor="UiO", ensure_exists=True)
```

Comment:
Won't `ensure_exists=True` crash the application with no recourse if we cannot create the expected directory?